BacktraceScreen.c

```diff
@@ -0,0 +1,301 @@
+#include "BacktraceScreen.h"
+
+#include <execinfo.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/types.h>
+#include <ctype.h>
+#include <unistd.h>
+
+#include "FunctionBar.h"
+#include "InfoScreen.h"
+#include "Panel.h"
+#include "Object.h"
+#include "Vector.h"
+#include "XUtils.h"
+#include "sys/wait.h"
+
+#define DWARF_INFORMATION_NOT_FOUND 1
+
+static const char* const BackTraceScreenFunctions[] = {"Refresh", "Done", NULL};
+static const char* const BackTraceScreenKeys[] = {"F3", "Esc"};
+static const int  BackTraceScreenEvents[] = {KEY_F(3), 27};
+
+static char *BacktraceScreen_parseEuStackFunction(char *current_line) {
+   while (current_line  && isspace(*current_line)) {current_line++;}
+   char *beginOfName = current_line;
+
+   char *backupOfLastSpace = current_line;
+   while (current_line && *current_line != '-') {
+      if (isspace(*current_line))
+         backupOfLastSpace = current_line;
+      current_line++;
+   }
+
+   size_t size_name = (backupOfLastSpace - beginOfName) + 1;
+   char *name = xStrndup(beginOfName, size_name);
+   name[size_name - 1] = '\0';
+   return name;
+}
+
+static char *BacktraceScreen_parseEuStackPath(char *current_line) {
```

Comment:
It could help debugging by adding a `const` qualifier since you never modify the string.

```suggestion
static char *BacktraceScreen_parseEuStackPath(const char *current_line) {
```